ts/Series/Flags/FlagsSeries.ts

```diff
@@ -526,25 +526,22 @@ class FlagsSeries extends ColumnSeries {
             );
 
             points.forEach(function (point): void {
-                const box = point.graphic && boxesMap[point.plotX as any];
-
-                if (box) {
-                    (point.graphic as any)[
-                        (point.graphic as any).isNew ? 'attr' : 'animate'
-                    ]({
-                        x: (box.pos as any) + (box.align as any) * box.size,
-                        anchorX: point.anchorX
-                    });
+                const plotX = point.plotX as number,
+                    box = point.graphic && boxesMap[plotX],
+                    graphic = point.graphic;
```

Comment:
If you assign `graphic` first, you can do `box = graphic && boxesMap[plotX]` and chip off two (!) bytes from the compiled code 😬